Add TestxServer, TestxController and TestxDirector documentation (#108)
aerogear/graphql-testx: CONTRIBUTING.md

# Contributing to GraphQL Testx

> First of all, thank you for your interest in contributing to the project

Reviewer: ```suggestion
Thank you for your interest in contributing to the GraphQL-TestX project
```

:heart_eyes::tada:

> The following is a set of guidelines for contributing to **GraphQL Testx**.

Reviewer: ```suggestion
Before contributing please read an set of the guidelines for contributing to **GraphQL Testx**:
```
